sdk/monitor/Azure.Monitor.OpenTelemetry/src/AzureMonitorOpenTelemetryExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Monitor.OpenTelemetry.Exporter;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry
+{
+    /// <summary>
+    /// Extension methods for setting up Azure Monitor OpenTelemetry in an <see cref="IServiceCollection" />.
+    /// </summary>
+    public static class AzureMonitorOpenTelemetryExtensions
+    {
+        /// <summary>
+        /// Adds Azure Monitor OpenTelemetry into service collection.
+        /// </summary>
+        /// <param name="services"><see cref="IServiceCollection"/>.</param>
+        /// <param name="configuration"><see cref="IConfiguration"/>.</param>
+        /// <returns><see cref="IServiceCollection"/></returns>
+        public static IServiceCollection AddAzureMonitorOpenTelemetry(this IServiceCollection services, IConfiguration configuration)
+        {
+            var options = new AzureMonitorOpenTelemetryOptions();
+            configuration.Bind(options);
+            return services.AddAzureMonitorOpenTelemetry(options);
+        }
+
+        /// <summary>
+        /// Adds Azure Monitor OpenTelemetry into service collection.
+        /// </summary>
+        /// <param name="services"><see cref="IServiceCollection"/>.</param>
+        /// <param name="options">The <see cref="AzureMonitorOpenTelemetryOptions" /> instance for configuration.</param>
+        /// <returns><see cref="IServiceCollection"/></returns>
+        public static IServiceCollection AddAzureMonitorOpenTelemetry(this IServiceCollection services, AzureMonitorOpenTelemetryOptions? options = null)
```

Comment:
Thinking out loud more....

How about:
`AddAzMonOtel(this services, bool enableTraces = true, bool enableMetrics = true, bool enableLogs = true);`

This can conditionally enable signals.
The rest of config (i.e part of azmonexporteroptions) we should retrieve from DI automatically.

Reply:
Wouldn't we want to let users control these signals without code change?

Reply:
I fear we might struggle if later there is new things we want to enable/disable (e.g. live metrics) if we have two places (explicit parameter vs. options object).